code/modules/paperwork/papershredder.dm

```diff
@@ -149,22 +156,21 @@
 		if(istype(P, /obj/item/flame/lighter/zippo))
 			class = "rose"
 
-		user.visible_message("<span class='[class]'>[user] holds \the [P] up to \the [src], it looks like \he's trying to burn it!</span>", \
-		"<span class='[class]'>You hold \the [P] up to \the [src], burning it slowly.</span>")
+		user.visible_message(span("[class]", "[user] holds \the [P] up to \the [src], it looks like \he's trying to burn it!"), \
```

Comment:
```suggestion
		user.visible_message(span("[class]", "[user] holds \the [P] up to \the [src], it looks like \he's trying to burn it!"), \
```

The text macro here could result in `they's` or `its''s`.

Maybe replace with : `\he [gender_datums[occupant.gender].is]`

Reply:
fixed by just removing it entirely